### ISSUE TITLE
Integrate Hexaly

### ISSUE THREAD

**Issue by Tobi-Huf** [rust-or/good_lp#57]
Hexaly is a new but rapidly rising solver in recent month that has outperformed some well-established solvers, such as Gurobi and CPLEX in terms of runtime. Is there a possibility to integrate Hexaly as a solver?

Thank you for considering this request.

**Comment by lovasoa** [comment]
Yes, it's probably possible... Please try and open a pull request!
